src/main/java/Vera.java

```diff
@@ -0,0 +1,245 @@
+import java.util.Scanner;
+
+public class Vera {
+    private static Task[] tasks;
+    private static int taskCount;
+    private static Scanner SCANNER;
+    private static final String PARTITION_LINE = "______________________________"
+            + "______________________________";
+
+    private static final String INVALID_INPUT_MESSAGE = "Please key in an appropriate command";
+    private static final String ERROR_MAX_TASK_MESSAGE = "Sorry! You've reached the maximum "
+            + "amount of tasks allowed on your task list";
+    private static final String ERROR_EVENT_MISSING_INPUT_MESSAGE = "Oops! You forgot to "
+            + "add a '/at' to your 'event' command";
+    private static final String ERROR_DEADLINE_MISSING_INPUT_MESSAGE = "Oops! You forgot to"
+            + " add a '/by' to your 'deadline' command";
+    private static final String ERROR_TODO_REPEATED_INPUT_MESSAGE = "Oops! It seems that you've "
+            + "already added this task.";
+
+    private static final int OPTIONS_INDEX = 0;
+    private static final int MARK_INDEX = 1;
+    private static final int TASK_CONTENT_INDEX = 1;
+    private static final int TASK_DESCRIPTION_INDEX = 0;
+    private static final int TASK_DESCRIPTION_INDEX_TODO = 1;
+    private static final int TASK_DATE_INDEX = 1;
+
+    public enum taskType {
+        TODO, DEADLINE, EVENT
+    }
+
+    public static void initTaskManager() {
+        tasks = new Task[100];
+        taskCount = 0;
+        SCANNER = new Scanner(System.in);
+    }
+
+    public static void printWithPartition(String message) {
+        System.out.println(PARTITION_LINE + System.lineSeparator() +
+                message + System.lineSeparator() + PARTITION_LINE);
```

Comment:
```suggestion
        System.out.println(PARTITION_LINE + System.lineSeparator()
                + message + System.lineSeparator() + PARTITION_LINE);
```
Maybe standardize the placement of the "+". I noticed sometimes your + is on the original line, other times it is on the next line. I feel putting the + on the next line makes it easier to read